hazelcast/src/test/java/com/hazelcast/config/AbstractConfigBuilderTest.java

```diff
@@ -0,0 +1,456 @@
+
+    @Test
+    public abstract void readLockConfig();
+
+    @Test
+    public abstract void readReliableTopic();
+
+    @Test
+    public abstract void readRingbuffer();
+
+    @Test
+    public abstract void readAtomicLong();
+
+    @Test
+    public abstract void readAtomicReference();
+
+    @Test
+    public abstract void readCountDownLatch();
+
+    @Test
+    public abstract void testCaseInsensitivityOfSettings();
+
+    @Test
+    public abstract void testManagementCenterConfig();
+
+    @Test
+    public abstract void testManagementCenterConfigComplex();
+
+    @Test
+    public abstract void testNullManagementCenterConfig();
+
+    @Test
+    public abstract void testEmptyManagementCenterConfig();
+
+    @Test
+    public abstract void testNotEnabledManagementCenterConfig();
+
+    @Test
+    public abstract void testNotEnabledWithURLManagementCenterConfig();
+
+    @Test
+    public abstract void testManagementCenterConfigComplexDisabledMutualAuth();
+
+    @Test
+    public abstract void testMapStoreInitialModeLazy();
+
+    @Test
+    public abstract void testMapConfig_minEvictionCheckMillis();
+
+    @Test
+    public abstract void testMapConfig_minEvictionCheckMillis_defaultValue();
+
+    @Test
+    public abstract void testMapConfig_preprocessingPolicy();
+
+    @Test
+    public abstract void testMapConfig_preprocessingPolicy_defaultValue();
+
+    @Test
+    public abstract void testMapConfig_evictions();
+
+    @Test
+    public abstract void testMapConfig_optimizeQueries();
+
+    @Test
+    public abstract void testMapConfig_cacheValueConfig_defaultValue();
+
+    @Test
+    public abstract void testMapConfig_cacheValueConfig_never();
+
+    @Test
+    public abstract void testMapConfig_cacheValueConfig_always();
+
+    @Test
+    public abstract void testMapConfig_cacheValueConfig_indexOnly();
+
+    @Test
+    public abstract void testMapStoreInitialModeEager();
+
+    @Test
+    public abstract void testMapStoreWriteBatchSize();
+
+    @Test
+    public abstract void testMapStoreConfig_writeCoalescing_whenDefault();
+
+    @Test
+    public abstract void testMapStoreConfig_writeCoalescing_whenSetFalse();
+
+    @Test
+    public abstract void testMapStoreConfig_writeCoalescing_whenSetTrue();
+
+    @Test
+    public abstract void testNearCacheInMemoryFormat();
+
+    @Test
+    public abstract void testNearCacheInMemoryFormatNative_withKeysByReference();
+
+    @Test
+    public abstract void testNearCacheEvictionPolicy();
+
+    @Test
+    public abstract void testPartitionGroupZoneAware();
+
+    @Test
+    public abstract void testPartitionGroupSPI();
+
+    @Test
+    public abstract void testPartitionGroupMemberGroups();
+
+    @Test
+    public abstract void testNearCacheFullConfig();
+
+    @Test
+    public abstract void testMapWanReplicationRef();
+
+    @Test
+    public abstract void testWanReplicationConfig();
+
+    @Test
+    public abstract void default_value_of_persist_wan_replicated_data_is_false();
```

Comment:
How about keeping one test method naming convention in this file and change it to something like: `testDefaultValueOfPersistWanRelicatedData()`?

I'm not sure if you have a method naming convention in the Core team, but I'd choose (it may be impossible for the whole project, then at least for this test file). You can check some generally accepted conventions here: https://dzone.com/articles/7-popular-unit-test-naming

Reply:
Renamed.

I kept the rest as is. There are cases with and without underscores, but I find both okay as long as the name is descriptive. Longer test names without separator are a bit hard to read.